src/ast/nodes/SwitchCase.ts

```diff
@@ -22,4 +24,16 @@ export default class SwitchCase extends NodeBase {
 		});
 		return addedNewNodes;
 	}
+
+	render (code: MagicString, options: RenderOptions) {
```

Comment:
Should this render function not have a third argument? Just wondering how much the polymorphism might bite us in performance as predictable singular argument signatures are important to optimizations.

Reply:
But for each SwitchCase, the signature is the same. As far as I know, call signatures are more important. Might be reasonable to consider adding "undefined" as a third parameter when calling render instead. But maybe we wait until we got rollup fit for the V8 web tooling benchmark so that we can talk to the actual experts here.

Reply:
Since a switch case is a statement I was thinking of a call site like the render call from `renderStatementList`, although I see this is distinguished by `needsBoundaries` now.